packages/website/src/components/code.tsx

```diff
@@ -1,14 +1,21 @@
-import { Code as BrightCode } from 'bright';
+import { getHighlighter } from 'shiki';
 
-import { ClientOnly } from './client-only.js';
 import theme from '../theme.json';
 
 type CodeProps = {
   code: string;
 };
 
-export const Code = ({ code, ...rest }: CodeProps) => (
-  <ClientOnly>
-    <BrightCode lang="tsx" theme={theme} code={code.trim()} {...rest} />
-  </ClientOnly>
-);
+export const Code = async ({ code, ...rest }: CodeProps) => {
+  const highlighter = await getHighlighter({
```

Comment:
You either cache the highlighter instance globally somewhere, use `import { codeToHtml } from 'shiki'` directly to avoid memory leak.

Reply:
That worked. Thanks! 🙂